types/mini-css-extract-plugin/index.d.ts

```diff
@@ -42,7 +42,7 @@ declare namespace MiniCssExtractPlugin {
         /**
          * Works like [`output.chunkFilename`](https://webpack.js.org/configuration/output/#outputchunkfilename).
          */
-        chunkFilename?: string;
+        chunkFilename?: Required<Configuration>['output']['chunkFilename'];
```

Comment:
Why `Required`?

Reply:
Forces `Configuration`'s optional fields like `output` to not be undefined in order to extract its child prop `chunkFilename` type

It's also how `filename` was defined above https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mini-css-extract-plugin/index.d.ts#L41